SingularityUI/app/templates/taskDetail/taskDeployFailureNotification.hbs

```diff
@@ -0,0 +1,44 @@
+{{#ifEqual deploy.deployResult.deployState "FAILED"}}
```

Comment:
if our task 404s you are not fetching the deploy info and are instead setting it to '', this if statement will throw 'has no attribute deployResult ...' type errors. Probably want to check that deploy is defined first